pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
 dynamic = ["version"]
 description = "Simulation based calibration and generation of synthetic data."
 dependencies = [
-  "pymc>=5.20",
+  "arviz>=0.20.0",
```

Comment:
This is ok for now. I added  `from_numpyro` https://github.com/arviz-devs/arviz-base/pull/39 (mostly copied code from old arviz and tweak it a little bit, most test are passing.)

Reply:
I think once it's merged in arviz-base, we can change it here.